webui/react/src/routes/index.ts

```diff
@@ -142,47 +135,12 @@ export const appRoutes: RouteConfig[] = [
 
 export const sidebarRoutes: RouteConfig[] = [
   dashboardRoute,
-  {
-    icon: 'experiment',
-    id: 'experiments',
-    path: '/ui/experiments',
-    title: 'Experiments',
-  },
-  {
-    icon: 'notebook',
-    id: 'notebooks',
-    path: '/ui/notebooks',
-    title: 'Notebooks',
-  },
-  {
-    icon: 'tensorboard',
-    id: 'tensorboards',
-    path: '/ui/tensorboards',
-    title: 'TensorBoards',
-  },
+  experimentListRoute,
+  taskListRoute,
   clusterRoute,
-  {
-    icon: 'shell',
-    id: 'shells',
-    path: '/ui/shells',
-    title: 'Shells',
-  },
-  {
-    icon: 'command',
-    id: 'commands',
-    path: '/ui/commands',
-    title: 'Commands',
-  },
 ];
 export const defaultSideBarRoute = sidebarRoutes[0];
 
-if (process.env.IS_DEV) {
```

Comment:
suggestion: maybe keep the code but make the list empty.

Reply:
good idea, in case we want to add future non-public pages